app/helpers/application_helper/toolbar/network_routers_center.rb

```diff
@@ -15,41 +15,43 @@ class ApplicationHelper::Toolbar::NetworkRoutersCenter < ApplicationHelper::Tool
             t,
             :klass => ApplicationHelper::Button::NetworkRouterNew
           ),
-          separator,
-          button(
-            :network_router_edit,
-            'pficon pficon-edit fa-lg',
-            t = N_('Edit selected Router'),
-            t,
-            :url_parms => 'main_div',
-            :enabled   => false,
-            :onwhen    => '1'
-          ),
-          button(
-            :network_router_add_interface,
-            'pficon pficon-edit fa-lg',
-            t = N_('Add Interface to selected Router'),
-            t,
-            :url_parms => "main_div",
-            :enabled   => false,
-            :onwhen    => "1"),
-          button(
-            :network_router_remove_interface,
-            'pficon pficon-edit fa-lg',
-            t = N_('Remove Interface from selected Router'),
-            t,
-            :url_parms => "main_div",
-            :enabled   => false,
-            :onwhen    => "1"),
-          button(
-            :network_router_delete,
-            'pficon pficon-delete fa-lg',
-            t = N_('Delete selected Routers'),
-            t,
-            :url_parms => 'main_div',
-            :confirm   => N_('Warning: The selected Routers and ALL of their components will be removed!'),
-            :enabled   => false,
-            :onwhen    => '1+'),
+          # TODO: Uncomment until cross controllers show_list issue fully in place/also supports? must be in place
+          # https://github.com/ManageIQ/manageiq/pull/12551
```

Comment:
this PR has been long merged.
Is there any javascript logic, that enables/disables those buttons?

Reply:
Yes it was, but the code is commented like this in most of the table toolbars. I don't think we can uncomment it until we solve the ```supports?``` there, since now I am getting exceptions in unsupported providers, because of missing methods. (AFAIK there is no JS logic for table level buttons&support mixin)

Reply:
ok, I dont really understand whats needed from the support mixin here. Maybe @ZitaNemeckova and @romanblanco could chime in, if this is a shortcoming in the supports_feature_mixin